webonary-cloud-api/lambda/db.ts

```diff
@@ -0,0 +1,43 @@
+export interface EntryFile {
+  id: string;
+  src: string;
+  fileClass?: string;
+  caption?: string;
+}
+
+export interface EntryValue {
+  lang: string;
+  value: string;
+  type?: string;
+}
+
+export interface EntrySense {
+  definitionOrGloss: EntryValue[];
```

Comment:
This will grow, but those changes can come later.

Reply:
of course...